.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'npm'
+          registry-url: 'https://repo.backbase.com/artifactory/api/npm/npm-backbase/'
+          scope: '@backbase'   
+      - name: Release
+        uses: ./.github/actions/bump-version
```

Comment:
Is it only about bumping versions or also about publishing packages?

Reply:
Later on we can add a step for publishing packages as well

Reply:
The semver package has a post-target hook to publish the version on NPM, GitHub or GitLab.
https://github.com/jscutlery/semver#triggering-executors-post-release